friend/controllers.py

```diff
@@ -4202,25 +4205,16 @@ def move_friends_to_another_voter(
                 status += "PROBLEM_UPDATING_FRIEND: " + str(e) + ' '
                 success = False
 
-    # delete -> this part ???
-    from_friend_list_remaining_results = friend_manager.retrieve_current_friend_list(
-        from_voter_we_vote_id,
-        read_only=False)
-    if not from_friend_list_remaining_results['success']:
-        status += from_friend_list_remaining_results['status']
-        success = False
-    from_friend_list_remaining = from_friend_list_remaining_results['current_friend_list']
-    for from_friend_entry in from_friend_list_remaining:
-        # Delete the remaining friendship values
-        try:
-            from_friend_entry.delete()
-        except Exception as e:
+    try:
+        number_deleted, details = CurrentFriend.objects\
+            .filter(viewer_voter_we_vote_id__iexact=from_voter_we_vote_id, )\
+            .delete()
+        from_voter_we_vote_id += number_deleted
+    except Exception as e:
             status += "PROBLEM_DELETING_FRIEND: " + str(e) + ' '
```

Comment:
I would recommend trying to get "ruff" working for VS Code so it can let you know when you are breaking PEP8 spacing rules.